js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedCodeLocationRow.tsx

```diff
@@ -30,54 +30,57 @@ const TEMPLATE_COLUMNS = '3fr 1fr 1fr 240px 160px';
 
 interface ErrorRowProps {
   locationNode: WorkspaceRepositoryLocationNode;
-  measure: (node: Element | null) => void;
+  index: number;
 }
 
-export const VirtualizedCodeLocationErrorRow = (props: ErrorRowProps) => {
-  const {locationNode, measure} = props;
-  const {name} = locationNode;
-  return (
-    <div ref={measure}>
-      <RowGrid border="bottom">
-        <RowCell>
-          <MiddleTruncate text={name} />
-        </RowCell>
-        <RowCell>
-          <div>
-            <LocationStatus location={name} locationOrError={locationNode} />
-          </div>
-        </RowCell>
-        <RowCell>
-          <div style={{whiteSpace: 'nowrap'}}>
-            <TimeFromNow unixTimestamp={locationNode.updatedTimestamp} />
-          </div>
-        </RowCell>
-        <RowCell>{'\u2013'}</RowCell>
-        <RowCell>
-          <JoinedButtons>
-            <ReloadButton location={name} />
-            <CodeLocationMenu locationNode={locationNode} />
-          </JoinedButtons>
-        </RowCell>
-      </RowGrid>
-    </div>
-  );
-};
+export const VirtualizedCodeLocationErrorRow = React.forwardRef(
+  (props: ErrorRowProps, ref: any) => {
```

Comment:
I should see if I can get a better `ref` type here.

Reply:
I think you want `React.ForwardedRef<HTMLDivElement>`?